_includes/pbc/all/install-features/202304.0/install-warehouse-user-management-feature.md

```diff
@@ -0,0 +1,319 @@
+| WarehouseUserAssignmentCollectionRequest        | class    | created    | src/Generated/Shared/Transfer/WarehouseUserAssignmentCollectionRequest        |
+| WarehouseUserAssignmentCollectionResponse       | class    | created    | src/Generated/Shared/Transfer/WarehouseUserAssignmentCollectionResponse       |
+| WarehouseUserAssignmentCollectionDeleteCriteria | class    | created    | src/Generated/Shared/Transfer/WarehouseUserAssignmentCollectionDeleteCriteria |
+| WarehouseUserAssignmentsRestAttributes          | class    | created    | src/Generated/Shared/Transfer/WarehouseUserAssignmentsRestAttributes          |
+| WarehousesRestAttributes                        | class    | created    | src/Generated/Shared/Transfer/WarehousesRestAttributes                        |
+| UserCollection                                  | class    | created    | src/Generated/Shared/Transfer/UserCollection                                  |
+| UserConditions                                  | class    | created    | src/Generated/Shared/Transfer/UserConditions                                  |
+| User.uuid                                       | property | created    | src/Generated/Shared/Transfer/User                                            |
+| User.isWarehouseUser                            | property | created    | src/Generated/Shared/Transfer/User                                            |
+| UserCriteria.userConditions                     | property | created    | src/Generated/Shared/Transfer/UserCriteria                                    |
+| StockCriteriaFilter.uuids                       | property | created    | src/Generated/Shared/Transfer/StockCriteriaFilter                             |
+| StockCriteriaFilter.stockIds                    | property | created    | src/Generated/Shared/Transfer/StockCriteriaFilter                             |
+| Collection                                      | class    | deprecated | src/Generated/Shared/Transfer/Collection                                      |
+| UserCriteria.idUser                             | property | deprecated | src/Generated/Shared/Transfer/UserCriteria                                    |
+| UserCriteria.email                              | property | deprecated | src/Generated/Shared/Transfer/UserCriteria                                    |
+| UserCriteria.userReference                      | property | deprecated | src/Generated/Shared/Transfer/UserCriteria                                    |
+| UserCriteria.withExpanders                      | property | deprecated | src/Generated/Shared/Transfer/UserCriteria                                    |
+| StockCriteriaFilter.idStock                     | property | deprecated | src/Generated/Shared/Transfer/StockCriteriaFilter                             |
+
+{% endinfo_block %}
+
+### 3) Set up configuration
+
+If you want to make `warehouse-user-assignments` resource protected, adjust the protected paths configuration:
```

Comment:
is this step optional?
If _yes_, then rewrite it as follows:
```suggestion
Optional: To make `warehouse-user-assignments` resource protected, adjust the protected paths configuration:
```
if _no_, then rewrite it as follows:
```suggestion
To make `warehouse-user-assignments` resource protected, adjust the protected paths configuration:
```